src/components/Items/ProfileContact.js

```diff
@@ -5,6 +5,7 @@ import PropTypes from 'prop-types';
 import MemoIc_Checklist from '../../assets/icons/Ic_Checklist';
 import {COLORS} from '../../utils/theme';
 import {normalize, normalizeFontSize} from '../../utils/fonts';
+import {colors} from '../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `colors` has been added, but the old import statement for `COLORS` is still present. If `COLORS` is no longer being used in this file, it should be removed to avoid confusion and unnecessary imports.
```diff
- import {COLORS} from '../../utils/theme';
+ import {colors} from '../../utils/colors';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->